src/briefcase/commands/base.py

```diff
@@ -479,17 +482,17 @@ def app_module_path(self, app):
             for path in app.sources
             if path.rsplit("/", 1)[-1] == app.module_name
         ]
-        try:
-            if len(app_home) == 1:
-                path = Path(str(self.base_path), *app_home[0])
-            else:
-                raise BriefcaseCommandError(
-                    f"Multiple paths in sources found for application '{app.app_name}'"
-                )
-        except IndexError as e:
+
+        if len(app_home) == 0:
```

Comment:
This was a bug; previously assumed to be caught as an IndexError, it was being returned as a `BriefcaseCommandError` with the wrong message.